README.md

```diff
@@ -1,50 +1,107 @@
 
-You can use the Common Web UI console to access information and features from other IBM Cloud Platform Common Services or IBM Cloud Paks that you install.
+You can use the ibm-commonui-operator to install the Common Web UI service for the IBM Cloud Platform Common Services and access the Common Web UI console. You can use the Common Web UI console to access information and features from other IBM Cloud Platform Common Services or IBM Cloud Paks that you install.
 
-For more information about the available IBM Cloud Platform Common Services and IBM Cloud Paks, see the [IBM Knowledge Center](http://ibm.biz/cpcsdocs).
+For more information about the available IBM Cloud Platform Common Services, see the [IBM Knowledge Center](http://ibm.biz/cpcsdocs).
 
 ## Supported platforms
 
-Red Hat OpenShift Container Platform 4.2 or newer installed on one of the following platforms:
+- Red Hat OpenShift Container Platform 4.2 or newer installed on one of the following platforms:
+
 - Linux x86_64
 - Linux on Power (ppc64le)
 - Linux on IBM Z and LinuxONE
 
 ## Operator versions
 
 - 1.1.0
+- 1.2.0
 
 ## Prerequisites
 
-The Common Web UI service depends on other IBM Cloud Platform Common Services. To install the Common Web UI operator (ibm-commonui-operator), you need to first install the operator dependencies and prerequisites:
+The Common Web UI service has dependencies on other IBM Cloud Platform Common Services. Before you install this operator, you need to first install the operator dependencies and prerequisites:
 
 - For the list of operator dependencies, see the IBM Knowledge Center [Common Services dependencies documentation](http://ibm.biz/cpcs_opdependencies).
 
 - For the list of prerequisites for installing the operator, see the IBM Knowledge Center [Preparing to install services documentation](http://ibm.biz/cpcs_opinstprereq).
 
-## SecurityContextConstraints Requirements
-
 ## Documentation
 
-For installation and configuration instructions if you are using the operator alone or with other common services, see the IBM Cloud Platform Common Services Knowledge Center [Installer documentation](http://ibm.biz/cpcsdocs).
+To install the operator with the IBM Common Services Operator follow the the installation and configuration instructions within the IBM Knowledge Center.
 
-If you are using the operator within an IBM Cloud Pak, see the documentation for the specific IBM Cloud Pak [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks)
+- If you are using the operator as part of an IBM Cloud Pak, see the documentation for that IBM Cloud Pak [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks).
+- If you are using the operator with an IBM Containerized Software, see the IBM Cloud Platform Common Services Knowledge Center [Installer documentation](http://ibm.biz/cpcs_opinstall).
 
-## Developer guide
+## SecurityContextConstraints Requirements
```

Comment:
What is the purpose of this section SecurityContextConstraints Requirements ?

Reply:
It is needed as part of the IBM Certification process. If all required sections are not in the readme, they will not certify us or we need to request exemption.

Reply:
Can title be: "Security Context Constraints

Reply:
According to the content playbook rules, it has to be the title I placed in the readme